package/exec-logrotate

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+function create_logrotate_config() {
+cat <<EOF > /etc/logrotate.conf
+/log/spdk_tgt.log {
+    rotate 1
+    size 20M
+    missingok
+    notifempty
+    copytruncate
+}
+
+/log/instance-manager.log {
+    rotate 1
+    size 20M
+    missingok
+    notifempty
+    copytruncate
+}
+EOF
+}
+
+# Create logrotate configuration
+create_logrotate_config
+
+# Like cron job, run logrotate every 1 minute
+while true; do
+    sleep 60
+    logrotate /etc/logrotate.conf
+done &
```

Comment:
I am thinking, can we add this to the crontab file instead?

Besides, is a 1-minute interval too short?

Reply:
> can we add this to the crontab file instead?

Any benefit?

> Besides, is a 1-minute interval too short?

logrotate checks if need to rotate log files, so the check is not heavy. 1 minute might be acceptable. WDYT?